eventually/team/models.py

```diff
@@ -31,37 +32,74 @@ class Team(models.Model):
         :param update_date: The date when
                     the certain team was last time edited.
         :type: create_date: datetime
+
+        :param owner: Describing relation between team and owner of team.
+        :type owner: int
+
+        :param members: Describing
+                            relations between team and members of team.
+        :type members: list
     """
     name = models.CharField(max_length=30)
     description = models.CharField(max_length=1024, blank=True)
     image = models.CharField(max_length=300, blank=True)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     updated_at = models.DateTimeField(auto_now=True)
+    owner = models.ForeignKey(CustomUser, null=True)
+    members = models.ManyToManyField(CustomUser, related_name='+')
```

Comment:
does `members` field need the `related name` keyword?

Reply:
yes, because I use CustomUser in ForeignKey and in ManyToManyField